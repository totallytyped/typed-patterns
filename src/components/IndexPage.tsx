// tslint:disable-next-line:no-unused-variable
import * as React from 'react';
import { Link } from 'react-router';


export default () =>
    <div>
        <div className="hero">
            <div className="wrap">
                <div className="text">
                    <strong>
                        Typed Patterns
                    </strong>
                </div>
                <div className="minitext">
                    Design Patterns from GoF book
                </div>
            </div>
        </div>
        <section className="content wrap">
            <p>
                Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
            </p>
            <section className="home-bottom-section">
                <div className="buttons-unit">
                    <Link className="button" to="/patterns">See more patterns</Link>
                </div>
            </section>
        </section>
    </div>;
