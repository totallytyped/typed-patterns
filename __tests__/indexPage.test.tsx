// tslint:disable-next-line:no-unused-variable
import * as React from 'react';
import IndexPage from '../src/components/IndexPage';
declare var require;
const renderer = require('react-test-renderer');

it('renders correctly', () => {
    const tree = renderer.create(<IndexPage />).toJSON();
    expect(tree).toMatchSnapshot();
});
