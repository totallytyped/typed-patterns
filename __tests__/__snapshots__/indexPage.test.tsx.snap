exports[`test renders correctly 1`] = `
<div>
  <div
    className="hero">
    <div
      className="wrap">
      <div
        className="text">
        <strong>
          Typed Patterns
        </strong>
      </div>
      <div
        className="minitext">
        Design Patterns from GoF book
      </div>
    </div>
  </div>
  <section
    className="content wrap">
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
    <section
      className="home-bottom-section">
      <div
        className="buttons-unit">
        <a
          className="button"
          onClick={[Function]}
          style={Object {}}>
          See more patterns
        </a>
      </div>
    </section>
  </section>
</div>
`;
